rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostTracker.java

```diff
@@ -104,6 +107,18 @@ protected void setNeedsNodes(boolean needNodesFlag) {
         this.needsNodesFlag.set(needNodesFlag);
     }
 
+    protected int addAndGetLostNodeNotification() {
+        return lostNodeNotificationsCounter.addAndGet(1);
+    }
+
+    protected void resetLostNodeNotifications() {
+        lostNodeNotificationsCounter = new AtomicInteger(0);
```

Comment:
I believe that this re-assigning might not work. Thus i think `lostNodeNotificationsCounter` should be volatile or just use `getAndSet(0)`